app/models/cart.rb

```diff
@@ -203,4 +203,9 @@ def total_price
       0.0
     end
   end
+
+  # may be replaced with paper-trail or similar at some point
```

Comment:
From an architectural stand point, I like that the version identifier could be swapped out easily (and per client, if needed). There is a concern that we'd switch to a new version system and effectively expire all pending approvals, so I'll add a comment to `Proposal#version` which calls out the backwards compatibility concerns.